app/views/trips/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<% render partial: "layouts/error_messages", locals: {model: @trip } %>
+
+<section class="trip_form">
+  <%= form_for @trip do |t| %>
+    <%= t.label :date %>
+    <%= t.date_field :date %>
+    <% unless @trip.passenger_id %>
```

Comment:
Watch the indentation here - since you're using a conditional, the contents inside should be indented